docs/configuration/CONFIGURATION_SUPPORTED_EXTRACTOR.md

```diff
@@ -219,8 +204,12 @@ export const MY_DEFAULT_CONFIG: MyConfig = {
 };
 ```
 
-Note that the order for the Nested config interface doesn't matter, but the default config needs to be put AFTER the
-interface declaration, and contain no variable references.
+> [!WARNING]
+> Default config needs to be put AFTER the interface declaration
+> and contain no variable references.
```

Comment:
Can you give an example on what not to do?

Reply:
I just understood you meant the placement in the file.
Maybe, you can replace put with written to make it more obvious